visualstudio/2017/netstandard/NetstandardModuleSample/NetstandardModuleSample/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "profiles": {
+    "NetstandardModuleSample": {
+      "commandName": "Executable",
+      "executablePath": "$(OutDir)gw.exe",
+      "commandLineArgs": "module_dev_sample.json",
+      "environmentVariables": {
+        "Key": "Value"
+      }
+    }
+  }
+}
```

Comment:
It will only happen when you create a new file. Next time, when you modify it, it will not remove the line ending.